ci/toolchains/TC-armv7-unknown-linux-gnueabihf.cmake

```diff
@@ -1,5 +1,4 @@
 set(CMAKE_SYSTEM_NAME Linux)
 set(CMAKE_SYSTEM_PROCESSOR armv7)
 set(ZENOHC_CUSTOM_TARGET armv7-unknown-linux-gnueabihf)
-set(CMAKE_C_COMPILER armv7-linux-gnueabihf-gcc)
-set(CMAKE_CXX_COMPILER armv7-linux-gnueabihf-g++)
+set(CMAKE_C_COMPILER armv7-linux-gnueabihf-gcc)
```

Comment:
same here, we need the cxx compiler entry

Reply:
Fixed. The reason I removed is that in the release workflow they failed to build when `CMAKE_CXX_COMPILER` was set.